### PR TITLE
Upgrade android on suzu

### PR DESCRIPTION
**Pull request by NeoTheThird**
see https://forums.ubports.com/topic/3229/sony-xperia-x-suzu-f5121

### CODE REVIEW

File: v1/suzu.json [hunk 1]
```diff
@@ -24,6 +24,11 @@
     "confirm_model": {
       "title": "Confirm your model",
       "description": "Please double-check that your device is a Sony Xperia X (F5121 or F5122)."
+    },
+    "upgrade_android": {
+      "title": "Upgrade Android",
+      "description": "If the device is running Android, you might have to upgrade to Android 8.0.",
```
Comment (Flohack74) [inline v1/suzu.json]: I would be more strict: The device needs to be on Android 8.0 before flashing, there is no _might_ option for that. See also original wording: _"Make sure to upgrade your device to the latest stock Android 8.0 before continuing"_ - Also, the word upgrade is dangerous, we need to ask @fredldotme but shouldn´t it rather be _to reset your device...._ to emphasize we want exactly that version?

Comment (fredldotme) [reply v1/suzu.json]: I think the wording is okay. People coming from SailfishOS probably already have the right bootloader installed. Also I think that "resetting" sounds more like factory reset than upgrading to the right Android version.

**Comment by fredldotme** [comment]
What we might want to add is an instruction on how to flash the right odm partition file before using the installer, either by adding the instructions in the installer or by linking to the forum post.

**Comment by Flohack74** [comment]
What if we just refer to the forum thread (or a Wiki page better,  in the forum you might get confused by other people´s postings)? Now we have information in multiple places, and its better to have a single source of truth (in case the truth changes lol)

**Comment by fredldotme** [comment]
Which place would be preferable for a wiki entry?

**Comment by NeoTheThird** [comment]
@fredldotme 
>What we might want to add is an instruction on how to flash the right odm partition file before using the installer, either by adding the instructions in the installer or by linking to the forum post.

we already do :) https://github.com/ubports/installer-configs/pull/16/files#diff-4324eac7f9c1b54626b6e33ddd2ec2b3R31

the upgrade section is not meant for long tutorials, that's why we have the link things there, so we can cover more grounds and have extensive external tutorials

**Comment by Flohack74** [comment]
> 
> 
> Which place would be preferable for a wiki entry?

Would be here: https://github.com/ubports/porting-notes/wiki

**Comment by NeoTheThird** [comment]
anyways, i'm merging this

**Pull request merged**